src/cloudsc_pyiface/src/pyiface/__init__.py

```diff
@@ -8,6 +8,5 @@
 # granted to it by virtue of its status as an intergovernmental organisation
 # nor does it submit to any jurisdiction.
 
-
-
-
+from pyiface.cloudsc_data import *  # noqa
+from pyiface.dynload import *  # noqa
```

Comment:
So should init.py stay empty, or it shouldn't ? I am confused with what I read in the net.

Reply:
Got it.